include/dts-functions.sh

```diff
@@ -1429,7 +1434,9 @@ main_menu_options(){
         fi
 
         # Use regular update process for everything else
-        ${CMD_DASHARO_DEPLOY} update
+        if ! ${CMD_DASHARO_DEPLOY} update; then
```

Comment:
`dasharo_deploy update` will return `1` in case of platform does not have updates available, and will trigger your `send_dts_logs ask`:

```bash
Enter an option:
2
Waiting for network connection ...
Network connection have been established!
Checking if board is Dasharo compatible.
Waiting for system clock to be synced ...
Current Dasharo version: 0.2.0
Latest available Dasharo version for your subscribtion: 0.2.0
No update available for your machine
Do you want to send console logs to 3mdeb? [N/y]: 
```

This is counterintuitive, could you fix it?

Reply:
85d646c6d5eeeb96a686f69a597077bd1f27a06a

Whole PR will need to be thoroughly tested

Reply:
force-pushed: 47c8d385650026bd0c82d4cfc66f1dbf55c16841, quick patch job, should probably think of something better. Maybe you have some ideas? If it works then I guess it could stay as is an be either fixed later on be made obsolete when we migrate to GO.